test/models/import/person_test.rb

```diff
@@ -20,4 +27,10 @@ class Import::PersonTest < ModelTestCase
     assert_equal 'CA', Import::Person.new(country: 'CAN').country_code
     assert_equal 'BA', Import::Person.new(country: 'BIH').country_code
   end
+
+  test 'medical_history' do
```

Comment:
Because this test is testing the class `Import::UpdateMedicalHistoryFromImport` it would be more appropriate to be in a new test file at `test/services/import/update_medical_history_from_import_test.rb`, this would follow the Rails file structure better. Would you mind moving it? Thanks!

Reply:
This test is great for ensuring that the class `Import::UpdateMedicalHistoryFromImport` is fulfilling it's duties, this test wouldn't ensure that the import is actually using this class to import the medical data. That's fine, as I think these pieces should be tested in isolation, but it might mean that adding another test would be worthwhile. We can chat about that if you have time!